wasm/lib/src/vm_class.rs

```diff
@@ -27,6 +27,7 @@ impl StoredVirtualMachine {
     fn new(id: String, inject_browser_module: bool) -> StoredVirtualMachine {
         let mut vm = VirtualMachine::new();
         let scope = vm.ctx.new_scope();
+        let _ = scope.init_builtins(&vm);
```

Comment:
Instead of ignoring the result, use `.unwrap()` instead. Then the error will be noticed earlier. The init_builtins function is to pass always, so this should not be a problem.